mod-invoice-storage/examples/voucher.sample

```diff
@@ -15,6 +15,10 @@
   "type": "Payment",
   "voucherDate": "2019-05-06T00:00:00.000+0000",
   "voucherNumber": "1000",
+  "acqUnitIds": [
```

Comment:
In schema `"id": "units"` so here in the sample, it should be `"units": [` ?

Reply:
It must match the name of the field, not the id.

Reply:
Then how will `"id"` be used?

Reply:
We don't use that. Just copy-pasted from orders 🙂

Reply:
🙂